python3/cloudwatch/sam-template.yaml

```diff
@@ -59,67 +60,23 @@ Outputs:
       Value: !GetAtt LogzioCloudwatchLogsLambda.Arn
 
 Resources:
-  LambdaPermission:
-    Type: 'AWS::Lambda::Permission'
-    Properties:
-      FunctionName: !GetAtt 
-        - LogzioCloudwatchLogsLambda
-        - Arn
-      Action: 'lambda:InvokeFunction'
-      Principal: !Join 
-        - .
-        - - logs
-          - !Ref 'AWS::Region'
-          - amazonaws.com
-      SourceAccount: !Ref 'AWS::AccountId'
-  LogzioSubscriptionFilter: 
-    Type: AWS::Logs::SubscriptionFilter
-    Properties: 
-      LogGroupName: !Ref LogGroup
-      FilterPattern: "LogziFilter"
-      DestinationArn: !GetAtt 
-        - LogzioCloudwatchLogsLambda
-        - Arn
-    DependsOn:
-      - LambdaPermission
-  lambdaIamRole:
-    Type: 'AWS::IAM::Role'
-    Properties:
-      RoleName: LogzioCloudWatchShipper
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Principal:
-              Service:
-                - lambda.amazonaws.com
-            Action:
-              - 'sts:AssumeRole'
-      Path: /
-      Policies:
-        - PolicyName: LogzioCloudWatchShipper
-          PolicyDocument:
-            Version: 2012-10-17
-            Statement:
-              - Effect: Allow
-                Action:
-                  - 'logs:PutResourcePolicy'
-                  - 'logs:CreateLogGroup'
-                  - 'logs:CreateLogStream'
-                  - 'logs:PutLogEvents'
-                Resource: '*'
   LogzioCloudwatchLogsLambda:
-    Type: AWS::Lambda::Function
+    Type: 'AWS::Serverless::Function'
     Properties:
       Runtime: python3.7
       Handler: lambda_function.lambda_handler
-      Code:
-        S3Bucket: logzio-aws-integrations-us-east-1
-        S3Key: cloudwatch-auto-deployment/0.0.2/logzio-cloudwatch.zip
+      CodeUri:
+        Bucket: logzio-aws-integrations-us-east-1
+        Key: cloudwatch-auto-deployment/0.0.2/logzio-cloudwatch.zip
```

Comment:
I think that since we've changed the template, and there are people that already use the `0.0.2` version, we should bump the version to `0.0.3`.
You'll also need to update the buckets according to it.